src/components/combo_box/combo_box.test.tsx

```diff
@@ -427,11 +427,9 @@ describe('behavior', () => {
       });
 
       act(() => {
-        const searchInputNode = searchInput.getDOMNode();
-        // React doesn't support `focusout` so we have to manually trigger it
-        searchInputNode.dispatchEvent(
-          new FocusEvent('focusout', { bubbles: true })
-        );
+        // React 16 failed on the previous `searchInputNode.dispatchEvent`
+        //  call that mocked a FocusEvent 'focusout'
+        searchInput.simulate('blur');
       });
```

Comment:
It looks like switching to `simulate` means the `act()` wrapper is no longer needed. I also don't think any of these comments are necessary at this point either
```suggestion
      searchInput.simulate('blur');
```